src/controller/artifact/annotation/parser.go

```diff
@@ -0,0 +1,76 @@
+package annotation
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/goharbor/harbor/src/lib/log"
+
+	"github.com/goharbor/harbor/src/pkg/artifact"
+)
+
+const (
+	// XIcon is icon content type image/x-icon
+	XIcon = "image/x-icon"
```

Comment:
Please remove the `x-icon` as the icon service introducing in #12683 only supports `gif`, `png` and `jpeg`